sympy/series/limits.py

```diff
@@ -124,6 +124,11 @@ def doit(self, **hints):
         from sympy.series.limitseq import limit_seq
 
         e, z, z0, dir = self.args
+        
+        if type(e) is exp:
+            for b in e.atoms(Symbol):
```

Comment:
I don't see why this is true. You're saying that if any symbol *anywhere* in the expression is zero then the whole thing is one.